responsiveness.css

```diff
@@ -0,0 +1,229 @@
+    color: #344563;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+  }
+
+  .header-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 21px;
+    width: 2%;
+  }
+
+  .flexcontainer {
+    width: 60%;
+    margin-top: 8%;
+    margin-left: 18%;
+    margin-bottom: 38%;
+    background-image: none;
+  }
+
+  .paragraph {
+    width: 84%;
+    margin-top: -5%;
+    margin-left: 6%;
+  }
+
+  .second {
+    margin-left: 8%;
+    margin-bottom: -5%;
+  }
+
+  .grid-container {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    width: 45%;
+    margin-left: 12%;
+    margin-bottom: 15%;
+    gap: 3%;
+  }
+
+  .about {
+    width: 14%;
+    margin-bottom: 5%;
+  }
+
+  .about-myself {
+    width: 53%;
+    margin-left: 1%;
+    margin-right: 1%;
+    margin-bottom: -5%;
+  }
+
+  .get-my-resume {
+    width: 10%;
+    margin-left: 3%;
+    margin-right: 3%;
+  }
+
+  .resume {
+    padding: 4px 5px 8px 7px;
+  }
+
+  .desktop-left {
+    width: 270px;
+    position: absolute;
+    height: 126px;
+    top: 370%;
+    right: 82%;
+  }
+
+  .desktop-right {
+    width: 260px;
+    height: 244px;
+    position: absolute;
+    top: 336%;
+    left: 84%;
+  }
+
+  .grid-container-another {
+    display: flex;
+    flex-direction: row;
+    margin-top: 5%;
+    margin-left: 13%;
+    margin-bottom: 6%;
+    width: 76%;
+  }
+
+  .grid-item-another {
+    width: 369px;
+    height: 316px;
+  }
+
+  .mini-flex-container-one {
+    margin-left: 15px;
+  }
+
+  .mini-flex-container,
+  .mini-flex-container-two {
+    justify-items: center;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .contact-section {
+    display: flex;
+    justify-content: center;
+    margin-left: -3%;
+    width: 105%;
+  }
+
+  .contact-text {
+    width: 30%;
+    height: 10%;
+    margin-left: 1%;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 52px;
+  }
+
+  .form-contact {
+    width: 684px;
+    height: 330px;
+    margin-left: 1%;
+  }
+
+  .form-item {
+    height: 48px;
+    width: 617px;
+    margin-left: -2%;
+  }
+
+  .message {
+    height: 114px;
+  }
+
+  .contact-button {
+    margin-left: 2%;
+  }
+
+  .desk-con-left {
+    position: absolute;
+    top: 505%;
+    right: 84%;
+    width: 236px;
+    height: 261.09px;
+  }
+
+  .desk-con-center {
+    position: absolute;
+    top: 520%;
+    right: 70%;
+    width: 105.27px;
+    height: 67.18px;
+  }
+
+  .desk-con-right {
+    position: absolute;
+    top: 490%;
+    left: 79%;
+    height: 452px;
+    width: 313.9015440419447px;
+  }
+
+  .footer {
+    margin-top: 25%;
+    margin-right: 15%;
+    margin-bottom: 3%;
+  }
+
+  .skills {
+    margin-left: 35%;
+  }
+}
```

Comment:
Please as I've suggested previously that you should follow the Figma design for the desktop version as much as possible, please kindly implement it for other aspects of your desktop version which also includes: the `About section` and `Contact section` . part of the problem you have with your code could be that you have several css files with conflicting class properties. You could try to have just one or two maximum css files and ensure that the properties are unique for each class 😄 

Figma design : 

https://github.com/tajulafreen/Portfolio/assets/20537712/2ab407ef-cbdc-4e22-928a-03b1011adecd

Your design :


https://github.com/tajulafreen/Portfolio/assets/20537712/b924d154-9e73-4304-a5c5-2cd3453d67c1